pkg/serverutils/knative_event_source_utils.go

```diff
@@ -0,0 +1,55 @@
+package serverutils
```

Comment:
Put this in `pkg/knative` instead of `pkg/serverutils`?
I think `pkg/serverutils` is intended for general utils.